bool forward
---
geometry_msgs/PoseStamped grasp
int16 index